### ISSUE TITLE
Fix OBO syntax to be compatible with OBO format version 1.4

### ISSUE THREAD

**Issue by althonos** [ebi-chebi/ChEBI#3867]
Hi!

[Two years ago](https://zenodo.org/record/3497420) I spent some time trying to improve the OBO syntax and semantics, and this resulted in the syntactical standardisation of a bunch of ontologies of the OBO Foundry. Basically, the new format version 1.4 has a lot more syntactical constraints that make it compatible with OWL2 and such. 

ChEBI itself is almost compatible with the newer [OBO format version 1.4](http://owlcollab.github.io/oboformat/doc/obo-syntax.html) except for a few `xref` clauses: since the Xref syntax was changed to make cross-references translated into RDF resources identifier instead of free text, their identifier cannot contain whitespaces, which is currently the case for 5 terms in the whole `chebi.obo` file:
- [ ] `CHEBI:49553`
- [ ] `CHEBI:84990`
- [ ] `CHEBI:133245`
- [ ] `CHEBI:16531`
- [ ] `CHEBI:77450`

Basically, fixing it means only escaping the whitespace with a backslash:
```ini
xref: PDBeChem:Copper(II) chloride   # before
xref: PDBeChem:Copper(II)\ chloride  # after
```

Hopefully your conversion / release script can be updated!

Cheers,
Martin

**Comment by amalik01** [comment]
The 5 ChEBI entries have now been fixed. Thanks for letting us know. The changes should be visible in the next ChEBI release.

**Comment by althonos** [comment]
Wonderful, thanks! This should also benefit many other ontologies from the OBO Foundry that are importing ChEBI.
